rs/execution_environment/src/scheduler.rs

```diff
@@ -810,19 +810,14 @@ impl SchedulerImpl {
                 .metrics
                 .round_inner_heartbeat_overhead_duration
                 .start_timer();
-            // Remove all remaining `Heartbeat`, `GlobalTimer`, and `OnLowWasmMemory` tasks
+            // Remove all remaining `Heartbeat`, `GlobalTimer` tasks
             // because they will be added again in the next round.
```

Comment:
```suggestion
            // Remove all remaining `Heartbeat` and `GlobalTimer` tasks
            // because they will be added again in the next round.
```